zipkin-ui/templates/index.mustache

```diff
@@ -10,7 +10,14 @@
       </select>
     </div>
 
-    <div id="time-stamp" class="form-group">
+    <div id="start-ts" class="form-group">
```

Comment:
no idea, but happy to change this and the dependency mustache post-merge, if you don't mind explaining on gitter what you are thinking